src/main/graphql/GetConsignment.graphql

```diff
@@ -2,5 +2,11 @@ query getConsignment($consignmentId: UUID!) {
     getConsignment(consignmentid: $consignmentId) {
```

Comment:
This query is used by existing queries which don't need the information about the file checks. GraphQL lets us be specific about which fields we fetch, so we should leave this query as it is and add a new one which just gets the information that will be required by the backend progress page.

Reply:
Would that be another getConsignment graphql file or within this same file?

Reply:
Also, would that mean it needs to be given another name or could it still be called getConsignment?

Reply:
Yeah, the name of the query in this project is actually independent of the name of the GraphQL query you're calling.

It's a bit like these two queries, which call the same GraphQL endpoint:

* https://github.com/nationalarchives/tdr-generated-graphql/blob/master/src/main/graphql/IsTransferAgreementComplete.graphql
* https://github.com/nationalarchives/tdr-generated-graphql/blob/master/src/main/graphql/GetTransferAgreement.graphql

Reply:
Ah cool, so different file and outer name but it's still utilising the actual getConsignment GraphQL query!